task/sanity-inspect-image/0.1/sanity-inspect-image.yaml

```diff
@@ -105,5 +106,6 @@ spec:
       BASE_IMAGE_REPOSITORY="$(jq -r '.Name | sub("[^/]+/"; "") | sub("[:@].*"; "")' "$BASE_IMAGE_INSPECT")"
       echo -n "$BASE_IMAGE_REPOSITORY" | tee $(results.BASE_IMAGE_REPOSITORY.path)
 
-      HACBS_TEST_OUTPUT="$(make_result_json -r SUCCESS -s 1)"
+      note="Task \`$(context.task.name)\` completed: Check inpected .json files under \`$(workspaces.source.path)/hacbs/$(context.task.name)\`."
```

Comment:
typo here on inpected. I assume that's supposed to be inspected?

Reply:
> 
Yes, thanks.